### DISCUSSION TITLE
mw3 -4 expire date - Do I need to update?

### DISCUSSION THREAD

**Discussion by michaeltz98** [mworion/MountWizzard4#214] [Q&A] [answered]
Hi,
I installed about a year ago the mw4 and now that I want to use it I see the "Delta T expires 2021-03-18" What is that mean?
do I need to update the software? if yes how?
I've used the mw3 for 2 years now last night the local time and time zone changed (I did not touch anything) can it be possible due to the mw3 expire date? Don't know how automatically change the local time and zone...
![Screen Shot 2022-01-10 at 22 02 04](https://user-images.githubusercontent.com/66162299/148831506-bce1a5f6-6495-4ee1-9944-6b0e58379413.png)

Thank you,
Michael

**Comment by mworion** [comment, marked as answer]
Hi Michael,

The deltaT data is related to earth rotation data used for correcting and adjusting the mount models. It could only predicted with a certain precision for some time an degrades in quality over time. MW shows you a hint if the update is long ago. The deltaT data only could be updated via the official 10 micron updater. MW help s you to download the actual data from an international source an automates the handling. There is no update for MW necessary. Even when the deltaT data degrades does not mean it’s unprecise. Depending on your setup you might not experience a difference in quality. The recommendation for keeping the data in a range for the best quality is to update this data 1-2 time a year. IERS as source updates the data on a daily base for research purpose.

Hope this helps.

Von meinem iPad gesendet

**Comment by michaeltz98** [comment]
If I see the delta - t icon on the keypad, this can affect satellite pointing? I saw that there was a great drift while tracking on them as well...

**Comment by mworion** [comment]
Yes, deviations of deltaT (or degradation) could lead into incorrect or worse pointing or tracking performance of the mount. As I do not know the internal details of the 10micron software I cannot tell you how much. To get the best performance (whatever that means absolutely) you need the latest Earth Rotation Data, a correct time and positions, actual and correct Refraction data and an actual pointing model of the mount. All these parameters are used to calculate and correct pointing and tracking.
